ddtrace/llmobs/_llmobs.py

```diff
@@ -829,8 +974,7 @@ def submit_evaluation(
         timestamp_ms = timestamp_ms if timestamp_ms else int(time.time() * 1000)
 
         if not isinstance(timestamp_ms, int) or timestamp_ms < 0:
-            log.warning("timestamp_ms must be a non-negative integer. Evaluation metric data will not be sent")
-            return
+            raise ValueError("timestamp_ms must be a non-negative integer. Evaluation metric data will not be sent")
```

Comment:
we should hold off changing this for the existing method since we're deprecating it anyway. It would just cause an apparent "breaking" change for users at this point

Reply:
Thanks for catching, didn't mean to alter `submit_evaluation`